src/upchunk.spec.ts

```diff
@@ -1,333 +1,322 @@
-import * as nock from 'nock';
-
+import { expect } from '@open-wc/testing';
```

Comment:
you're using expect to match what was here as closely as possible, right?

Reply:
yessir